layout/layout.go

```diff
@@ -559,7 +549,8 @@ func (i *Image) SaveAs(name string, additionalNames ...string) error {
 	}
 
 	var diagnostics []imgutil.SaveDiagnostic
-	err = path.AppendImage(i.Image)
+	annotations := imageRefAnnotation(additionalNames...)
```

Comment:
It's a bit confusing that we ignore the originally provided `name` here... are we still in the situation where `name` is the path and `additionalNames` are the annotations?

Reply:
Yeah, we were, but based on our talk I just changed the logic